File: Códigos/fluttersorting/lib/ui/about.dart
import 'package:flutter/material.dart';

class About extends StatelessWidget {
  final String title;
  const About({Key? key, required this.title}) : super(key: key);

  @override
  Widget build(BuildContext context) {
    return LayoutBuilder(
      builder: (_, constraints) => Material(
        color: Colors.white,
        child: Center(
          child: SingleChildScrollView(
            child: Column(
              crossAxisAlignment: CrossAxisAlignment.stretch,
              children: [
                const Text(
                  'Versão: Flutter',
                  textAlign: TextAlign.center,
                  style: TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
                ),
                Container(
                  margin: const EdgeInsets.only(top: 50),
                  child: const Text(
                    'Carine Casa Grande',
                    textAlign: TextAlign.center,
                    style: TextStyle(fontSize: 20),
                  ),
                ),
                Container(
                  margin: const EdgeInsets.fromLTRB(20, 50, 20, 0),
                  child: const Text(
                    'Comparação entre o desempenho de aplicações para smartphones desenvolvidos em Flutter e React Native:',
                    textAlign: TextAlign.center,
                    style: TextStyle(fontSize: 20, fontWeight: FontWeight.bold),
                  ),
                ),
                Container(
                  margin: const EdgeInsets.fromLTRB(20, 0, 20, 0),
                  child: const Text(
                    'Uma análise utilizando algoritmos de ordenação',
                    textAlign: TextAlign.center,
                    style: TextStyle(fontSize: 20),
                  ),
                ),
                Container(
                  margin: const EdgeInsets.only(top: 40, right: 20, left: 100),
                  child: const Text(
                    'Trabalho de Conclusão de Curso apresentado ao Curso de Ciência da Computação do Centro Universitário Filadélfia – UniFil, como requisito parcial à obtenção do título de Bacharel em Ciência da Computação.\n\nOrientadora: Profª. Simone Sawasaki Tanaka',
                    textAlign: TextAlign.justify,
                    style: TextStyle(fontSize: 20),
                  ),
                ),
                Container(
                  margin: const EdgeInsets.only(top: 40),
                  child: const Text(
                    'Londrina\n2021',
                    textAlign: TextAlign.center,
                    style: TextStyle(fontSize: 20),
                  ),
                ),
                Padding(
                  padding: const EdgeInsets.fromLTRB(20, 30, 20, 10),
                  child: SizedBox(
                    height: 53,
                    child: ElevatedButton(
                      onPressed: () {
                        Navigator.pop(context);
                      },
                      child: const Text(
                        'Voltar',
                        style: TextStyle(
                            color: Colors.white,
                            fontSize: 20,
                            fontWeight: FontWeight.bold),
                      ),
                      style: ButtonStyle(
                        backgroundColor: MaterialStateProperty.all<Color>(
                          const Color.fromRGBO(241, 100, 31, 1),
                        ),
                        shape:
                            MaterialStateProperty.all<RoundedRectangleBorder>(
                          RoundedRectangleBorder(
                            borderRadius: BorderRadius.circular(10),
                          ),
                        ),
                      ),
                    ),
                  ),
                ),
              ],
            ),
          ),
        ),
      ),
    );
  }
}

File: Códigos/sortingreactnative/src/assets/style/about.js
import {StyleSheet} from 'react-native';

export default StyleSheet.create({
  content: {
    padding: 20,
    justifyContent: 'center', // Eixo primário (Y)
    alignItems: 'center', // Eixo secundário (X)
  },
  title: {
    fontWeight: 'bold',
    fontSize: 20,
    textAlign: 'center',
    marginTop: 10,
  },
  subtitle: {
    fontSize: 20,
    textAlign: 'center',
    marginTop: 40,
  },
  titlePaperwork: {
    fontWeight: 'bold',
    fontSize: 20,
    textAlign: 'center',
    marginTop: 40,
  },
  subtitlePaperwork: {
    fontSize: 20,
    textAlign: 'center',
  },
  description: {
    marginTop: 40,
    fontSize: 20,
    textAlign: 'justify',
    marginLeft: 50,
  },
  location: {
    textAlign: 'center',
    marginTop: 40,
    fontSize: 20,
  },
  button: {
    backgroundColor: '#f1641f',
    padding: 18,
    borderRadius: 10,
    alignSelf: 'stretch',
    marginTop: 20,
    marginBottom: 10,
  },
  buttonText: {
    textAlign: 'center',
    color: '#ffffff',
    fontSize: 20,
    fontWeight: 'bold',
  },
});
